README.md

```diff
@@ -1,5 +1,15 @@
+[![Stars](https://img.shields.io/github/stars/laminlabs/bionty?logo=GitHub&color=yellow)](https://github.com/laminlabs/bionty)
+[![pypi](https://img.shields.io/pypi/v/bionty?color=blue&label=pypi%20package)](https://pypi.org/project/bionty)
+
 # bionty: Basic biological entities
 
 bionty is documented & tested within lamindb: github.com/laminlabs/lamindb.
 
+- Access public ontologies of biological entities
```

Comment:
This here should be exactly the same text as here:
<img width="732" alt="image" src="https://github.com/user-attachments/assets/e448a980-1989-4d70-b2df-67424ffdeacd">

And it should be harmonized with what we have on the "features" page. Of course, it should be more detailed here than there.
<img width="747" alt="image" src="https://github.com/user-attachments/assets/fb50c354-e834-4029-b83b-0f966059c634">

For instance, it's important to stress that its >20 ontologies, and that it supports ontology versioning and ontology extension.